### PR TITLE
scdoc: Pen.schelp a few minor improvements

### PR DESCRIPTION
**Pull request by redFrik**
hid some private methods, adjusted layout for capStyle example, SCPopUpMenu -> PopUpMenu, added missing ::

### CODE REVIEW

**Comment by telephon** [comment]
if you wanted, you could make a PR for 3.9, as it is a bugfix. But fine as it is, too.

**Comment by telephon** [comment]
this is so annoying - one can't see in advance how the pr will turn out when changing the destination.

what bedt to do?

**Comment by redFrik** [comment]
i have no idea what's going on.  i 100% used the web interface to first edit the helpfile and do the initial pr.  then i clicked 'edit' and changed base from develop to 3.9.  i expected this to work.  shall i just close and redo it all the long-winded way?

**Comment by brianlheim** [comment]
> what's best to do?

rebase onto 3.9, then force push.

> i 100% used the web interface to first edit the helpfile and do the initial pr. then i clicked 'edit' and changed base from develop to 3.9. i expected this to work. shall i just close and redo it all the long-winded way?

you edited the version of the file that was on develop. if you want to use the web interface, change the branch you're viewing to 3.9 before you edit.

**Comment by patrickdupuis** [comment]
I think the issue is that your branch is based on `develop`. Essentially, you are trying to merge `develop` + `redfrik-patch1` into `3.9`. You would need to rebase your branch on `3.9` in order to get rid of the commits that are part of `develop`. I'm not certain if this can be done from within the web interface. I'm looking into it now.

**Comment by patrickdupuis** [comment]
@brianlheim is there a way to rebase once the PR has been created? `refrik-pach1` is a change that was made from within the web interface, not a local branch.

**Comment by patrickdupuis** [comment]
ok :)

**Comment by patrickdupuis** [comment]
Can we merge this PR into `develop` then, so that the work isn't lost?

**Comment by brianlheim** [comment]
I am in favor of doing that. @snappizz are you ok with that? Then we'll resolve and merge https://github.com/supercollider/supercollider/pull/3224, then merge develop into 3.9, and everything will be peachy.

**Comment by brianlheim** [comment]
Now that dev has been merged into 3.9, the piggybacking commits are gone. Sorry for the stress everyone!

File: HelpSource/Classes/Pen.schelp [hunk 1]
```diff
@@ -632,12 +632,15 @@ Draw a bitmap image using the Image class.
...
 code::
+(
 w = Window.new.front;
 w.view.background = Color.red;
 i = Image.open(SCDoc.helpSourceDir +/+ "images/Swamp.png");
 w.drawFunc_({
 	Pen.drawImage( Point(140, 140), i, operation: 'sourceOver', opacity:1);
 });
+)
+::
```
Comment (brianlheim) [inline HelpSource/Classes/Pen.schelp]: wow, thanks for that!

**Pull request merged**